scripts/sip/sip_interaction.py

```diff
@@ -452,3 +452,23 @@ def createProposalSIP0049():
     print(datas)
     print(description)
     # createProposal(contracts['GovernorOwner'], targets, values, signatures, datas, description)
+
+# SIP to set the max vesting withdraw iterations in staking contract
+def createProposalSIP0050(maxWithdrawIterations):
```

Comment:
`maxWithdrawIterations` should already be in the SIP  
pls remove the param and assign an explicit value to the data

Reply:
Can you provide me the link of the SIP? could not find any

Reply:
SIP-050 text should have this parameter value listed so that the users would see the changes by the SIP explicitly  
therefore no need to have it as a parameter in the SIP proposal creation script (this function) - just set it as a literal value which should be the same as in the SIP - SIP text TBD

Reply:
Updated